sklearn/metrics/_classification.py

```diff
@@ -1243,6 +1303,8 @@ def fbeta_score(
     ``true positive + false negative == 0``, f-score returns 0 and raises
     ``UndefinedMetricWarning``. This behavior can be
     modified with ``zero_division``.
+    Note that if zero_division is np.nan, such values will be excluded
+    from the average.
```

Comment:
Same here, I think we can place this in the `zero_division` description.

Similar comment for the other docstrings.